mobile/src/fingerprint-popup/fingerprint-popup.android.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import FingerprintScanner from 'react-native-fingerprint-scanner';
+import { FingerprintPopupStyle } from './fingerprint-popup.style';
+import { Image, StyleProp, Text, TextStyle, TouchableOpacity, View, ViewStyle } from 'react-native';
+import Modal from 'react-native-modal';
+
+//============================================================================
+interface FingerprintPopupState {
+    error: boolean;
+}
+
+//============================================================================
+interface FingerprintPopupProps {
+    isVisible: boolean;
+    onPopupClose: () => void;
+    onPopupHidden: (success: boolean) => void;
+}
+
+//============================================================================
+export class FingerprintPopupAndroid extends Component<FingerprintPopupProps, FingerprintPopupState> {
+    private success = false;
+
+    //----------------------------------------------------------------------------
+    constructor(props: FingerprintPopupProps) {
+        super(props);
+
+        this.handleAuthenticationAttempt = this.handleAuthenticationAttempt.bind(this);
```

Comment:
use instead of bind:
```typescript
private handleAuthenticationAttempt = (error: Error): void => {
...
}
```

Reply:
Addressed in 29b70b8f7989b0d930725c2c79041cf00c4c8d12